src/server/handlers/static.go

```diff
@@ -39,6 +41,16 @@ func (s *Static) SetNext(h http.Handler) {
 func (s Static) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	path := s.MediaPath + r.URL.EscapedPath()
 
+	// Check for ".." in the url path,
+	// if we find ".." in the path we will not serve static files
+	doubleDotRegex := regexp.MustCompile("[.]{2,}")
+	if doubleDotRegex.MatchString(path) {
+		log.Printf("Found double dot in static path\n")
+
+		s.next.ServeHTTP(w, r)
```

Comment:
I personally would suggest returning "Not Found" as it's obviously an attempt to attack.
But unless you're sure passing it to the next middleware is OK, I am fine with this.